common/static/js/spec/utility_spec.js

```diff
@@ -16,3 +16,25 @@ describe('utility.rewriteStaticLinks', function() {
         ).toBe('<img src="http://www.mysite.org/static/foo.x"/>');
     });
 });
+describe('utility.rewriteCdnLinksToStatic', function() {
+    'use strict';
```

Comment:
Is it necessary to use `use strict`?

Reply:
Yes, function form of 'use strict' is required here to avoid the eslint error.